make-seccomp-filters.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+set -ufe
+>&2 echo THIS IS A DEVELOPER SCRIPT
+>&2 echo YOU DO NOT NEED TO RUN IT UNLESS YOU EDITED seccomp-bpf.c
```

Comment:
This script could (and should) be wired into `meson.build` to ensure the `seccomp-bpf.h` is up-to-date? This way we could even remove and gitignore the `seccomp-bpf.h` altogether.

Reply:
Your two comments are related, so I'll bunch them together:

I opted not to change any behavior at all in this PR, so I submitted stuff matching the existing build system, hence a separate script for this (previously, it was a completely isolated depgraph in make)

The reason meson doesn't generate this script for us is because it'd add a dep on seccomp-bpf on the build machine (this is a problem outside of Linux, or even for particularly pedantic Linux users).

My personal choice for replacing this (as a less conservative change than what I was doing) would be to write the BPF by hand or to simply require seccomp-bpf at runtime.

Another option that was mentioned is dropping it altogether, as we don't really have any daemons here that run at high privileges anyway.

`use_seccomp` being a boolean rather than a feature is indeed since it doesn't dictate a dependency (hence auto makes little sense for it). I wanted to replace this with a check for -EINVAL at runtime but, again, decided to make everything as similar to before the change as humanly possible while still changing stuff.